spec/workers/podcast_episodes/bust_cache_worker_spec.rb

```diff
@@ -0,0 +1,26 @@
+require "rails_helper"
+
+RSpec.describe PodcastEpisodes::BustCacheWorker, type: :worker do
+  describe "#perform" do
+    let!(:podcast) { create(:podcast) }
+    let!(:podcast_episode) { FactoryBot.create(:podcast_episode, podcast_id: podcast.id) }
```

Comment:
This can be moved inside a context since, they are only used in the spec that checks with a valid episode, also you can replace `let!` with `let`, there's no need to create them at each spec.

Something like:

```ruby
    context "when podcast episode is found" do
      let(:podcast) { create(:podcast) }
      let(:podcast_episode) { create(:podcast_episode, podcast_id: podcast.id) }

      it "busts cache" do
```

Reply:
ok, I was about to do that. I will move it inside a context